target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/Makefile

```diff
@@ -2,12 +2,12 @@
 # Makefile for the Atheros AR71xx built-in ethernet macs
 #
 
-ag71xx-y	+= ag71xx_main.o
-ag71xx-y	+= ag71xx_gmac.o
-ag71xx-y	+= ag71xx_ethtool.o
-ag71xx-y	+= ag71xx_phy.o
+ag71xx_ds-y	+= ag71xx_main.o
+ag71xx_ds-y	+= ag71xx_gmac.o
+ag71xx_ds-y	+= ag71xx_ethtool.o
+ag71xx_ds-y	+= ag71xx_phy.o
 
-ag71xx-$(CONFIG_AG71XX_DEBUG_FS)	+= ag71xx_debugfs.o
+ag71xx_ds-$(CONFIG_AG71XX_LEGACY_DEBUG_FS)	+= ag71xx_ds_debugfs.o
 
-obj-$(CONFIG_AG71XX)	+= ag71xx_mdio.o
-obj-$(CONFIG_AG71XX)	+= ag71xx.o
+obj-$(CONFIG_AG71XX_LEGACY)	+= ag71xx_ds_mdio.o
+obj-$(CONFIG_AG71XX_LEGACY)	+= ag71xx_ds.o
```

Comment:
For some parts you added `legacy` and for others you added `ds`. I would prefer the same extension everywhere. I would even prefer `legacy` if it does not get too long.

Reply:
renamed module name to legacy